modules.txt

```diff
@@ -0,0 +1,780 @@
+# github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
```

Comment:
Self-Reminder: Delete this file... not sure where it came from but there is one in the vendor folder that is modified; suspect that's the intended one.